### DISCUSSION TITLE
How to activate profile owner ?

### DISCUSSION THREAD

**Discussion by adryano85** [BinTianqi/OwnDroid#22] [Show and tell]
Hello ,I have a Xiaomi 13T pro with Hyperos and a can't activate profile owner ,how to do this setting?

**Comment by BinTianqi** [comment]
Did you try Shizuku? 

Take a look of your accounts in system settings, make sure that there isn't any account before you activate OwnDroid. 

_"accounts"_ is your accounts in android system, not the Xiaomi account! 

Activate a profile owner in main user is not recommeded. Why not try device owner? 

> You can create a work profile with OwnDroid if you don't want to clear your accounts.

**Reply by adryano85** [reply]
How to activate a profile owner in Xiaomi ,please give a link video

**Comment by adryano85** [comment]
> Did you try Shizuku? 
> 
> Take a look of your accounts in system settings, make sure that there isn't any account before you activate OwnDroid. 
> 
> _"accounts"_ is your accounts in android system, not the Xiaomi account! 
> 
> Activate a profile owner in main user is not recommeded. Why not try device owner? 
> 
> > You can create a work profile with OwnDroid if you don't want to clear your accounts. 

Or how to create a device owner in Xiaomi ?

**Comment by BinTianqi** [comment]
Do you have Shizuku installed on your device?

Install Shizuku and go to "Deactivated -> Shizuku" then grant shizuku permission and activate device owner

**Reply by adryano85** [reply]
I made this thing...error 255

**Reply by adryano85** [reply]
I made this thing...error 255

**Comment by BinTianqi** [comment]
First, open _usb debugging(security)_ in developer options. 

Second, remove all accounts

**Reply by adryano85** [reply]
But I don't want to remove Google accounts

**Comment by BinTianqi** [comment]
It's impossible to activate device owner or profile owner in main user if some accounts exist.

**Reply by Minoricew** [reply]
It's possible now, use HookDPM.

**Reply by nktnet1** [reply]
Link for reference:

- https://github.com/BinTianqi/HookDPM

Requires root and Xposed.

**Reply by BinTianqi** [reply]
Just received a notification from this 2-year-old thread. Note that HookDPM is no longer under active maintainence.
